docs/development/local_setup.md

```diff
@@ -68,9 +70,63 @@ Currently there are no binary builds available, but it is fairly simple to build
 You can verify the installation by running following command:
 
     ```console
-    $ etcdbrctl -v
-    INFO[0000] etcd-backup-restore Version: v0.7.0-dev
-    INFO[0000] Git SHA: 38979f0
-    INFO[0000] Go Version: go1.12
-    INFO[0000] Go OS/Arch: darwin/amd64
+    ➜ etcdbrctl -v
+    INFO[0000] etcd-backup-restore Version: v0.29.0-dev     
+    INFO[0000] Git SHA: 0247d8d4                            
+    INFO[0000] Go Version: go1.22.1                         
+    INFO[0000] Go OS/Arch: darwin/arm64
     ```
+
+## Running `etcdbrctl` as a process
+
+> [!WARNING]  
+> Ensure that you do not add your `credentialDirectory` or `credential_file.json` to your version control. Easiest way to ensure this, is to avoid using `git add .`. You could add these files/directory as `.gitignore` targets, or have the credential file/directory exist in a directory which is not tracked by `git`.
+
+### Environment variables
+
+Taking the example of S3 here, one has to:
+
+* Set the `AWS_APPLICATION_CREDENTIALS` environment variable if the credentials are to be stored in a directory, where each file in the directory holds a different credential
+* Set the `AWS_APPLICATION_CREDENTIALS_JSON` if the credentials are to be stored in `.json` file.
+
+Export the following environment variables according to your host OS:
+
+* `ETCD_CONF=example/01-etcd-config.yaml`
+* `POD_NAME=POD_0`
+* `POD_NAMESPACE=POD_NAMESPACE_0`
+* One of the following based on the authentication type:
+  * `AWS_APPLICATION_CREDENTIALS=/path/to/your/credential/file/directory/s3credentials`
+  * `AWS_APPLICATION_CREDENTIALS_JSON=/path/to/your/credential/file/credential_file.json`
+
+> For example: `export ETCD_CONF=example/01-etcd-config.yaml`, etc. on Unix based OS-es, and using the `set` command in Windows.
+
+`example/01-etcd-config.yaml` has the configuration required to set up `etcdbrctl` while running `etcd` as a process alongside `etcdbrctl`.
```

Comment:
A small suggestion, I feel it would be better to say explicitly that, this config is a dummy of the actual configmap which is required by `etcd` process. Currently it is not clear as to who uses that out of the two processes. This is assuming the reader of this doc is someone new to `etcdbr`

Reply:
Alright, will add that!
This ConfigMap is mounted into the `etcd-backup-restore` container in the `Etcd` pod which gives information about the `etcd` container to `etcd-backup-restore`.